src/utils/isWeb.ts

```diff
@@ -1,3 +1,5 @@
-export default typeof window !== 'undefined' &&
-  typeof window.HTMLElement !== 'undefined' &&
-  typeof document !== 'undefined';
+const UNDEFINED = 'undefined';
```

Comment:
Would you save more bytes by using `let`? 😆

```suggestion
let UNDEFINED = 'undefined';
```